### PR TITLE
Update risco documentation

### PR DESCRIPTION
**Pull request by pergolafabio**
## Proposed change

Additional info to use service accounts with risco integration

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/risco.markdown [hunk 1]
```diff
@@ -14,6 +14,10 @@ ha_quality_scale: platinum
 ---
 
 This integration connects with Risco Alarms over [Risco Cloud](https://riscocloud.com/).
+<div class='note warning'>
+
+Recommendation: Create a new regular account on www.riscocloud.be with the same roles as you use with the Android/Iphone app. Only one session can do writes at a time, other users can only read.
```
Comment (OnFreund) [inline source/_integrations/risco.markdown]:
```suggestion
It is recommended to use  a regular (non-owner) account with the Risco app/website, and use a different regular account with the integration. Risco has restrictions on concurrent uses by different users, especially if they have different permission levels.
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
Please use the term `iPhone` instead of `Iphone` 😉

**Review by OnFreund — DISMISSED** [review]

**Review by klaasnicolaas — DISMISSED** [review]
✅ Approved. Can be merged as soon as checks are finished 👍

File: source/_integrations/risco.markdown [hunk 2]
```diff
@@ -15,6 +15,12 @@ ha_quality_scale: platinum
 
 This integration connects with Risco Alarms over [Risco Cloud](https://riscocloud.com/).
 
+<div class='note warning'>
+
+  It is recommended to use  a regular (non-owner) account with the Risco app/website, and use a different regular account with the integration. Risco has restrictions on concurrent uses by different users, especially if they have different permission levels.
+
+</div>
+
```
Comment (frenck) [inline source/_integrations/risco.markdown]:
```suggestion
<div class='note warning'>
It is recommended to use a regular (non-owner) account with the Risco app/website, and use a different regular account with the integration. Risco has restrictions on concurrent uses by different users, especially if they have different permission levels.
</div>

```

**Review by frenck — APPROVED** [review]

**Pull request merged**